src/lib/rate-limiter.ts

```diff
@@ -6,7 +6,7 @@ import createHttpError from 'http-errors';
 import type { ExtendedContext } from '../types.js';
 import { credits } from './credits.js';
 
-const redisClient = createPersistentRedisClient({ legacyMode: true });
+const redisClient = createPersistentRedisClient();
```

Comment:
Looks like it may be `getPersistentRedisClient` now.

Reply:
👍